config/projects/chefdk.rb

```diff
@@ -65,6 +65,7 @@
 # Going to leave 2.1.5 for now since there is a workaround
 override :'ruby-windows', version: "2.1.6"
 override :'ruby-windows-devkit', version: "4.7.2-20130224-1151"
+override :'openssl-windows', version: "1.0.1m"
```

Comment:
chef uses openssl 1.0.0x. The default version remains so that that will work correctly.